isis/appdata/import/MerMI.tpl

```diff
@@ -0,0 +1,201 @@
+{% extends "img_base.tpl" %}
+
+{% block instrument %}
+    {% if exists("ROVER_MOTION_COUNTER") %}
+    {% set motionCounter=ROVER_MOTION_COUNTER.Value %}
+    RoverMotionCounter          = ({% for count in motionCounter %}
+                                    {% if loop.is_last %}
+                                    {{ count }}
+                                    {% else %}
+                                    {{ count }},
+                                    {% endif %}
+                                   {% endfor %})
```

Comment:
is this if/else block needed

Reply:
It is for determining if a comma should be added after a number. For example, the desired output is `(1, 2, 3, 4, 5)`. So the if statement is checking if it's the last element, if not add a comma.